tests/saml/test_configuration.py

```diff
@@ -0,0 +1,202 @@
+# FIXME: Required to get rid of the circular import error
+import api.app
```

Comment:
Can you go into more detail on this? Do you need to import api.app inside the test because putting the import statement where you'd prefer to put it would cause a circular import?

Reply:
I had some weird errors while trying to run tests in specific files, this import was required to resolve those issues but it's not required anymore. I removed all these redundant imports